doc/api/fs.md

```diff
@@ -685,6 +685,33 @@ added: v10.0.0
 
 Changes the ownership of a file.
 
+### `fsPromises.copy(src, dest[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `src` {string|Buffer|URL} source path to copy.
+* `dest` {string|Buffer|URL} destination path to copy to.
+* `options` {Object|Function}
+  * `dereference` {boolean} dereference symlinks. **Default:** `false`.
```

Comment:
Not blocking but I prefer something like `followSymLinks` for clarity

Reply:
@jasnell perhaps you could give feedback on the naming discussion above with @targos, if we go the route of naming this method `cp`, I have a preference for keeping options close to the unix `cp` command.

Reply:
Understood. Like I said, not blocking at all. :-)